lambda-runtime/src/AWS/Lambda/Client.hs

```diff
@@ -0,0 +1,175 @@
+module AWS.Lambda.Client
+  ( HTTPConfig
+  , InternalLambdaClientError (..)
+  , LambdaClient
+  , LambdaResponse (..)
+  , getHttpConfig
+  , getNextLambdaResponse
+  , sendBootError
+  , sendEvent
+  )
+where
+
+import AWS.Prelude
+import Control.Exception (displayException)
+import Control.Monad.Except (ExceptT(..), liftEither, throwError, withExceptT)
+import System.Environment (lookupEnv)
+
+import qualified Data.Aeson          as JSON
+import qualified Data.ByteString     as BS
+import qualified Network.HTTP.Client as HTTP
+import qualified Network.HTTP.Types  as HTTP
+
+data HTTPConfig = HTTPConfig
+  { request :: HTTP.Request
+  , manager :: HTTP.Manager
+  }
+
+data LambdaResponse = LambdaResponse
+  { event     :: JSON.Value
+  , requestId :: RequestId
+  }
+  deriving stock Show
+
+newtype RequestId = RequestId Text
+  deriving stock (Show)
+
+instance Conversion BS.ByteString RequestId where
+  convert (RequestId requestId) = encodeUtf8 requestId
+
+data InternalLambdaClientError
+  = LambdaContextDecodeError Text
+  | MissingLambdaRunTimeApi
+  | InvalidLambdaRunTimeApi Text
```

Comment:
Not blocking but we should sort these constructors.

Reply:
Just FY I have sorted this constructor.  Just not sure why GH is not showing this comment as out-dated (addressed)